ee/allowedcmd/cmd_linux.go

```diff
@@ -13,10 +13,12 @@ func Apt(ctx context.Context, arg ...string) (*exec.Cmd, error) {
 	return validatedCommand(ctx, "/usr/bin/apt", arg...)
 }
 
+var ErrHomebrewNotFound = errors.New("homebrew not found")
```

Comment:
I am wondering if we would instead want to move this to `cmd.go`, rename it something more generic like `ErrCommandNotFound` and return it from the `validatedCommand` function -- this new error is very useful here, and I imagine we might want to do similar in the future when other commands aren't found. 

(This will also have the bonus side effect of fixing the build errors, since we run `Brew` on both macOS and Linux so this new error needs to be available to darwin builds as well.)

Reply:
That's great idea, I'll update the PR with the changes.